packages/metadata-common/src/AddMetadataButton.tsx

```diff
@@ -110,14 +106,26 @@ export const AddMetadataButton = (
             <MenuList id="split-button-menu">
               {sortedSchema?.map((schema: IDictionary<any>) => (
                 <MenuItem
-                  key={schema.display_name}
-                  title={`New ${schema.display_name} ${props.schemaType ?? ''}`}
+                  key={schema.title}
+                  title={`New ${schema.title} ${
+                    schema.title
+                      .toLowerCase()
+                      .includes(props.schemaType?.toLocaleLowerCase())
+                      ? ''
+                      : props.schemaType
```

Comment:
I don't understand this syntax.  What ends up being the value of `title` here?  Oh, I see. So its saying:

_add the value of `schemaType` to the end of `title` unless the value of `schemaType` is already in `title`_

That seems reasonable and follows the behavior that `schemaType` is truly _additional_ text.

Reply:
Correct. 
This is where we render the options of creating new metadata for multi-schema schemaspaces after clicking the + button.
Sometimes the schema title may already contain that extra info (eg. "Filesystem Component Catalog"), so in that case we avoid repetition of the additional text (which would be "component catalog"), but necessary otherwise (eg. "Apache Airflow runtime configuration").